main.py

```diff
@@ -525,31 +525,54 @@ def delete_tag_range(self, request: Request) -> JSONResponse:
             raise HTTPException(400, detail=str(err))
         return JSONResponse("Operation Successful", status_code=200)
 
+    @rest('v3/interfaces/{interface_id}/special_tags', methods=['POST'])
+    @validate_openapi(spec)
+    def set_special_tags(self, request: Request) -> JSONResponse:
+        """Set special_tags"""
+        content_type_json_or_415(request)
+        content = get_json_or_400(request, self.controller.loop)
+        tag_type = content.get("tag_type")
+        special_tags = content["special_tags"]
+        interface_id = request.path_params["interface_id"]
+        interface = self.controller.get_interface_by_id(interface_id)
+        try:
+            interface.set_special_tagss(tag_type, special_tags)
```

Comment:
Related to https://github.com/kytos-ng/kytos/pull/431/files#r1409657038, this should be `set_special_tags`, right?

Reply:
Corrected